app/assets/stylesheets/partials/anime-page.css.scss

```diff
@@ -701,4 +701,3 @@
     line-height: 38px;
   }
 }
-
```

Comment:
General policy on Unix is to end all files in `\n` so it could be that your editor decided to remove that.  I recommend installing an EditorConfig plugin for your editor or trying a different editor